proto/internal/temporal/server/api/persistence/v1/chasm.proto

```diff
@@ -61,11 +61,24 @@ message ChasmComponentAttributes {
         temporal.api.common.v1.DataBlob data = 4;
         // Versioned transition of the Entity when the task was created.
         VersionedTransition versioned_transition = 5;
+        // The xth task generated in this versioned transition.
+        // Together with the versioned transition, this is a unique identifier for
+        // this task.
+        int64 versioned_transition_offset = 6;
+        // If a physical task is created for this task in this cluster.
+        // NOTE: this is a cluster-specific field and can not be replicated.
+        // Changes to this field also doesn't require an increase in versioned transition. 
+        int32 physical_task_status = 7;
```

Comment:
I could use a bool for task status, but int is more flexible and it's var encoded anyway.